src/services/users.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let users = [];
+
+function getAll() {
+  return [...users];
+}
+
+function getById(id) {
+  if (!id) {
+    return null;
+  }
+
+  const foundUser = users.find(user => String(user.id) === id);
+
+  return foundUser || null;
+}
+
+function create(name) {
+  const newUser = {
+    id: users.length + 1,
```

Comment:
When creating a new user, generate a unique id (e.g. using uuid library) instead of using users.length + 1.
Or create function to find max id & plus one to this value

Reply:
Done